### ISSUE TITLE
Switch Pro Controller not working

### ISSUE THREAD

**Issue by toyota90** [Hydr8gon/VitaControl#18]
hi team, i have a psp 1000 vita, and seems after install of the plug in, the pro controller connects the vita via bluetooth but only the Home button works, no other buttons work, if someone is able to walk me through the steps of what i can do to solve this would be good. I have attempted to follow the instruction previously advised by others in the other chat and it doesnt work. Looking forward to a reply with a solution

**Comment by ikarusdt** [comment]
hi, seems like a common issue, I use a pro controller, once it has paired make a reset to the console, once it has made the reset push once the pairing button in the pro controller and you should see it working correctly. or at least it works for me :P

**Comment by Xenchip** [comment]
**TL:DR Solution on the very last paragraph. With the bold text**.

> hi team, i have a psp 1000 vita, and seems after install of the plug in, the pro controller connects the vita via bluetooth but only the Home button works, no other buttons work, if someone is able to walk me through the steps of what i can do to solve this would be good. I have attempted to follow the instruction previously advised by others in the other chat and it doesnt work. Looking forward to a reply with a solution.

At first, I did, and it didn't work. But then by following what @ikarusdt tell on here

> hi, seems like a common issue, I use a pro controller, once it has paired make a reset to the console, once it has made the reset push once the pairing button in the pro controller and you should see it working correctly. or at least it works for me :P

I did reset the console, and once it was pairing to, no controller button can be working. It does connect to the Bluetooth. So, I went to turn on the H-encore Application to turn on the Henkaku Mode then pressing the pairing button and randomly hit any button, it does work all smooth. I have done not find any trouble at this moment. Thanks for Hydr8gon that create this tool. It is very useful tool when the ps vita needed and button is broken like I did instead of replacing the button. 

Updates: The controller has no issues, but when the device is restarted or shutdown and turned on again it will be trying to pair the controller again to connect. After retrying so many times, trying to connect this, is a little tricky. Sometimes it works and sometimes it doesn't. The simple way is don't restart the ps vita to save time. Just turn on and off the Bluetooth and pairing the nintendo switch controller, press any button on the switch button between the end of pair time when the Bluetooth on ps vita is done pairing and must not be too late. When it does not connect, try turn off the Bluetooth, delete the pro controller Profile settings on Bluetooth controller then try again pair the new pro controller profile until it can connect. For my experience, the switch controller and ps vita Bluetooth pairing must be pair at the same time because after retrying to connect and ps vita doesn't find the controller it will disconnect the switch controller, and if the switch controller button was pressed it will be trying to be pairing the device again. Once switch controller any button pressed it will be trying to re-pair. So, simple way to know when the time to pair is by turn and off the Bluetooth and pair at the same time until it does connect (additionally I have on another situation charged the controller trying to connect and it can connect, but logically it must be because the device pairing simultaneously).

**Re-Updates:** After trying more than hundreds of times. The issues still persist, the controller does not recognize sometimes. Annoying part from my experience was the first time it was paired before other devices, and if it was disconnected from vitas then paired by other devices. **Solution is simply, make sure Bluetooth should have already paired from vita to make the controller works then try press any button if it doesn't work do rebooting the Vita and try to reconnect it, if it still doesn't work turn on and turn off the Bluetooth until switch controller able to connect and repeat to do step on rebooting until it connects (From my experience, plug to charge the controller and unplug it may increase the chance to connect). Try to press any button following by the green light then pressing arrow or analog button, the light sometimes drops to one green indicator light even if it was connected on vitas it wasn't actually connected, try turn on and off the Bluetooth again and repeat to hit the button until the green light indicator blinking from one light indicator to another light indicator, then hit the arrow or analog button again, if it was moving properly then it worked.**

**Comment by zjhken** [comment]
> hi, seems like a common issue, I use a pro controller, once it has paired make a reset to the console, once it has made the reset push once the pairing button in the pro controller and you should see it working correctly. or at least it works for me :P

I've tried the reset way, at first it works. However, in the second time when I tried to play the reset trick, it didn't work. Not easy to find the consistent way.
